python/amico/core.py

```diff
@@ -51,8 +52,9 @@ def __init__( self, study_path, subject ) :
 
         self.set_config('peaks_filename', None)
         self.set_config('doNormalizeSignal', True)
+        self.set_config('doKeepb0Intact', False)  # does change b0 images in the predicted signal
         self.set_config('doComputeNRMSE', False)
-
+        self.set_config('doSaveCorrectedDWI', False)
```

Comment:
I don't think it would break other models, right? I'm just using them in the Freewater part...